src/Host.h

```diff
@@ -56,16 +56,13 @@ class dlgNotepad;
 class TMap;
 
 
-class Host  : public QObject
-{
+class Host : public QObject {
     friend class XMLexport;
     friend class XMLimport;
 
 public:
-
-                       Host( int port, const QString& mHostName, const QString& login, const QString& pass, int host_id );
-
-                       ~Host();
+    Host(int port, const QString& mHostName, const QString& login, const QString& pass, int host_id);
```

Comment:
This is the down side to not aligning all the function/method and members names - the constructors and destructor loose all indentation but I guess we have to live with it - don't we? 😞